proto/src/universe.proto

```diff
@@ -25,10 +25,19 @@ message Zone {
     string name = 2;
 }
 
+message KeyRange {
+    // This should be left unset in CreateKeyspaceRequest.
+    string base_range_uuid = 1;
```

Comment:
I think maybe let's just remove this? The creator of the keyspace doesn't really care what the range ids are. The warden can assign the ids for the base ranges itself

Reply:
I was thinking the Universe implementation would persist the base ranges and would assign a uuid at that point. That way Warden can be stateless for base ranges and can still reuse prior assignments (reading from database) on restart.

Reply:
Hmm, I am still inclined to think the universe shouldn't be concerned about range ids. The warden can have a DB table that maps keyspace_id -> base_ranges for example, and when it sees a new keyspace from the universe it (statelessly) creates the base ranges and writes them to the table (using a cassandra conditional write on them not existing before to avoid races). WDYT?

Reply:
If you still prefer the Universe creates the keys, then maybe let's just have two different protos one with ids and one without?

Reply:
Yeah, it still feels like Universe should own creating primary keys for the base ranges. But I have now split out the protos like you suggested.